go-runtime/compile/schema.go

```diff
@@ -322,27 +320,29 @@ func goNodePosToSchemaPos(node ast.Node) (schema.Position, int) {
 	return schema.Position{Filename: p.Filename, Line: p.Line, Column: p.Column, Offset: p.Offset}, fset.Position(node.End()).Column
 }
 
-func visitGenDecl(pctx *parseContext, node *ast.GenDecl) error {
+func visitGenDecl(pctx *parseContext, node *ast.GenDecl) {
 	switch node.Tok {
 	case token.TYPE:
 		if node.Doc == nil {
-			return nil
+			return
 		}
-		directives, err := parseDirectives(fset, node.Doc)
+		directives, err := parseDirectives(node, fset, node.Doc)
 		if err != nil {
-			return err
+			pctx.errors = append(pctx.errors, err)
 		}
 		for _, dir := range directives {
 			switch dir.(type) {
 			case *directiveExport:
 				if len(node.Specs) != 1 {
-					return errorf(node, "error parsing ftl export directive: expected exactly one type "+
-						"declaration")
+					pctx.errors = append(pctx.errors, errorf(node, "error parsing ftl export directive: expected "+
+						"exactly one type declaration"))
+					return
 				}
 				if pctx.module.Name == "" {
 					pctx.module.Name = pctx.pkg.Name
 				} else if pctx.module.Name != pctx.pkg.Name {
```

Comment:
Unrelated to this change but I don't think this is correct - it should be more like this:
```suggestion
				} else if strings.HasPrefix(pctx.pkg.Name, "ftl/" + pctx.module.Name) {
```

Though this is also not quite correct because for module "foo" it would match `ftl/foo` and `ftl/foobar`.

Reply:
how about `if strings.TrimPrefix(pctx.pkg.Name, "ftl/") == pctx.module.Name`?

Reply:
actually, won't the pctx.pkg.Name here just be the module name without the `ftl` prefix?